docs/editor/profiles.md

```diff
@@ -179,52 +179,50 @@ VS Code comes with a predefined set of profile templates that you can use to cus
 The Python profile is a good starting point for Python development. It comes with Python specific snippets and has the following extensions:
 
 * [autoDocstring](https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring) - Generate Python docstrings automatically.
-* [Black Formatter](https://marketplace.visualstudio.com/items?itemName=ms-python.black-formatter) - Formatting support using the [black](https://github.com/python/black) formatter.
 * [Docker](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-docker) - Create, manage, and debug containerized applications.
-* [Even Better TOML](https://marketplace.visualstudio.com/items?itemName=tamasfe.even-better-toml) - Fully-featured TOML support.
-* [Python](https://marketplace.visualstudio.com/items?itemName=ms-python.python) - IntelliSense, linting, formatting, debugging, refactoring.
+* [Even Better TOML](https://marketplace.visualstudio.com/items?itemName=tamasfe.even-better-toml) - Fully-featured TOML support for e.g. pyproject.toml files.
```

Comment:
Maybe we can add the <code>`</code> with pyproject.toml to highlight this is the file.

```diff
- Fully-featured TOML support for e.g. pyproject.toml files.
+ Fully-featured TOML support for e.g. `pyproject.toml` files.
```